benchmarks/benchmarks/stats.py

```diff
@@ -56,7 +56,97 @@ def time_ttest_ind_diff_var(self):
         stats.ttest_ind(self.a, self.c, equal_var=False)
 
 
+class DistributionsAll(Benchmark):
+    # all distributions are in this list. A conversion to a set is used to
+    # remove duplicates that appear twince in either `distcont` or
+    # `distdiscrete`.
+    dists = sorted(list(set([d[0] for d in distcont + distdiscrete])))
+
+    param_names = ['distribution', 'method']
```

Comment:
```suggestion
    param_names = ['dist_name', 'method']
```
I think this makes the distinction between `dist` and `distribution` more obvious.
Not a big deal, of course, but it's easier for us to do it than for a future maintainer to have to figure it out.